package-lock.json

```diff
@@ -5923,9 +5923,9 @@
       "dev": true
     },
     "cypress": {
-      "version": "3.3.1",
-      "resolved": "https://registry.npmjs.org/cypress/-/cypress-3.3.1.tgz",
-      "integrity": "sha512-JIo47ZD9P3jAw7oaK7YKUoODzszJbNw41JmBrlMMiupHOlhmXvZz75htuo7mfRFPC9/1MDQktO4lX/V2+a6lGQ==",
+      "version": "3.4.1",
```

Comment:
Cypress is supposed to be installed through the `@vue/cli-plugin-e2e-cypress`dependency, but it wasn't updated ? How is it possible ?

What commands did you used make these updates ?